layers/meta-resin-raspberrypi/recipes-kernel/linux-firmware/linux-firmware_git.bbappend

```diff
@@ -0,0 +1,14 @@
+# This .txt file is required for the firmware to work. It used to be
+# shipped, but now it's ommitted.
+#
+# XXX 2017.02.27 We believe this to be a bug, and the code below is a workaround
```

Comment:
Are you talking about the XXX comment? I added it for the future. This way, when the bug is fixed upstream, we can scan the code every once in a while for "XXX" and see if the issues still persist. Otherwise, in a couple of years, our codebase will have accumulated a lot of cruft.